src/types/avatar.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  AvatarAccessoriesType,
+  AvatarClotheColor,
+  AvatarClotheType,
+  AvatarEyebrowType,
+  AvatarEyeType,
+  AvatarFacialHairColor,
+  AvatarFacialHairType,
+  AvatarGraphicType,
+  AvatarHairColor,
+  AvatarMouthType,
+  AvatarSkinColor,
+  AvatarTopType,
+} from "constants/avatar";
+
+export interface AvataaarProps {
+  // possibly look into separating accentColor from the other properties at some point
+  // otherwise don't forget to omit when needed
+  accentColorClass: string;
+  skinColor: AvatarSkinColor;
+  topType: AvatarTopType;
```

Comment:
Maybe to make it more descriptive, call it "hairStyle"?

Reply:
Same as in https://github.com/inovex/scrumlr.io/pull/4270#discussion_r1705414372